### PR TITLE
Datashaper Update

### PR DESCRIPTION
**Pull request by darthtrevino**

### CODE REVIEW

File: javascript/app-model-exposure/src/profile/ExposureResource.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+			definitions: this.definitions,
+			question: this.question,
+			selectedTableName: this.selectedTableName,
+		}
+	}
+
+	public override loadSchema(schema: ExposureResourceSchema | undefined) {
+		super.loadSchema(schema)
+		this.title = schema?.title ?? this.defaultTitle()
+		console.log("LEODHTNS", schema, schema?.title, schema?.name, this.defaultTitle(), this.defaultName())
```
Comment (monik182) [inline javascript/app-model-exposure/src/profile/ExposureResource.ts:64]: is this log intentional?

File: javascript/app-model-exposure/src/profile/ExposureAppRoot.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+			loadState={loadState}
+		>
+			<ModelExposurePage href={href} />
+		</RecoilBasedProfileHost>
+	)
+})
+
+function loadState(resource: ExposureResource, { set }: MutableSnapshot) {
+	set(projectNameState, resource.projectName)
+	set(causalFactorsState, resource.causalFactors)
```
Comment (monik182) [inline javascript/app-model-exposure/src/profile/ExposureAppRoot.tsx:37]: in the current branch I"m working on I added a map to associate an id for the factors, which fixes an error we currently have in the dev site where the table is not being correctly populated. I did the same for the definitions. So my question is, in this version the table pop bug is not happening?

File: javascript/app-model-exposure/src/profile/ExposureResource.ts (hunk 1 above)
Comment (darthtrevino) [reply javascript/app-model-exposure/src/profile/ExposureResource.ts]: Nope, debugging a hang

File: javascript/app-model-exposure/src/profile/ExposureAppRoot.tsx (hunk 2 above)
Comment (darthtrevino) [reply javascript/app-model-exposure/src/profile/ExposureAppRoot.tsx:37]: I'm not sure TBH, can you demonstrate the issue for me tomorrow?

Comment (darthtrevino) [reply javascript/app-model-exposure/src/profile/ExposureAppRoot.tsx]: Per our discussion, including a patch in the loadData function

**Pull request merged**